### PR TITLE
Adding login feature

### PR DESCRIPTION
**Pull request by lqueipom**

### CODE REVIEW

**Comment by Tetheta** [comment]
Starting review

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    if (userService.isUserLoggedIn()) {
+      String email = userService.getCurrentUser().getEmail();
+      String urlToRedirectToAfterLogOut = "/login";
+      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterLogOut);
+
+      response.getWriter().println("<p>Hello " + email + "!</p>");
+      response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
+
+    } else {
+      String urlToRedirectToAfterLogIn = "/data";
```
Comment (Tetheta) [inline portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:28]: Did you push your change? looks like this is still pointing to "/data"

**Comment by Tetheta** [comment]
Oh this was requested before we figured that out, lmk when you push that change

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java (hunk 1 above)
Comment (lqueipom) [reply portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java]: Oh thought I had that's strange, will do again :)

Comment (lqueipom) [reply portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java]: Hi Travis! It shows as if it is already updated to me, could you check again? (the change was made in the "fixing log-in redirect" commit)

**Comment by Tetheta** [comment]
Sorry for the delay, didn't get an email fro some reason.

**Pull request merged**